enhancements/cvo/object-removal-manifest-annotation.md

```diff
@@ -0,0 +1,196 @@
+---
+title: use-manifest-annotation-for-object-removal
+authors:
+  - "@jottofar"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@wking"
+  - "@smarterclayton"
+  - "@deads2k"
+approvers:
+  - "@smarterclayton"
+  - "@deads2k"
+creation-date: 2020-08-03
+last-updated: 2020-08-10
+status: implementable
+---
+
+# Use Manifest Annotation For Object Removal
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Currently CVO managed object removal is handled by jobs.
+Jobs have many immutable components, so they aren't a great match for CVO managed manifests.
+This enhancement replaces that approach with a new manifest annotation requesting the CVO delete the in-cluster object instead of creating/updating it.
+This will provide a more straightforward way for engineering to remove content.
+
+## Motivation
+
+There should be a straightforward way to remove CVO managed objects.
+
+### Goals
+
+* Developers will be able to remove any of the currently managed CVO objects by modifying an existing manifest and adding the new delete annotation.
+
+### Non-Goals
+
+* This enhancement does not add any new objects for management by CVO, therefore the new delete annotation only applies to the currently supported manifests and any that may be added in the future.
+This enhancement neither adds nor requires any application specific delete logic within CVO.
```

Comment:
This sentence seems like it wants to be a separate list entry (`* This enhancement neither ...`), because application-specific logic seems orthogonal to growing new type support.